### PR TITLE
[RCD-42] log-configs/cluster.yaml: Include debug messages in json log

### PR DESCRIPTION
**Pull request by rvl**
## Description

We need debug-level logging from nodes in the cluster.

## Linked issue

https://iohk.myjetbrains.com/youtrack/issue/RCD-42

## Type of change

- Bug fix of configuration file.

## QA Steps
```
nix-build -A cardano-sl-node-static -o cardano-node
./cardano-node/bin/cardano-node-simple --address 127.0.0.1:3000 --listen 127.0.0.1:3000 --system-start `date +%s` --log-config log-configs/cluster.yaml --logs-prefix tmp/logs --db-path tmp/db --configuration-file lib/configuration.yaml --configuration-key mainnet_dryrun_full --topology scripts/bench/topology/topology.yaml --node-id node0
# check tmp/logs/node.json for sev: Debug
```

### CODE REVIEW

File: log-configs/cluster.yaml [hunk 1]
```diff
@@ -6,13 +6,11 @@ rotation:
...
   severity: Debug+
-  files:
-    - node.log
 
   handlers:
     - { name: "JSON"
       , filepath: "node.json"
       , logsafety: SecretLogLevel
-      , severity: Info
+      , severity: Debug+
```
Comment (jbgi) [inline log-configs/cluster.yaml]: do we need to specify the severity here, as it seems to be the same as the default?

File: log-configs/cluster.yaml [hunk 2]
```diff
@@ -6,13 +6,11 @@ rotation:
 
 loggerTree:
   severity: Debug+
-  files:
```
Comment (jbgi) [inline log-configs/cluster.yaml]: Does this means there will be no flat logs anymore? is that ok?

Comment (rvl) [reply log-configs/cluster.yaml]: There will be no more `node.log`, which is good. There are also flat logs written to stdout/stderr, which are captured in the systemd-journal.

File: log-configs/cluster.yaml (hunk 1 above)
Comment (rvl) [reply log-configs/cluster.yaml]: Good point - probably not, but I think it's useful to explicitly set the value.

**Pull request merged**